test/e2e/storage/testsuites/volume_io.go

```diff
@@ -44,13 +44,13 @@ import (
 	admissionapi "k8s.io/pod-security-admission/api"
 )
 
-// MD5 hashes of the test file corresponding to each file size.
+// sha256 hashes of the test file corresponding to each file size.
```

Comment:
While we used fnv for other non-cryptographic hashing purposes, here we want to be able to compute the hash in the pod, and sha256 has a commandline already available in our test image.

Sha256 should be plenty fast for this purpose anyhow.